python/lsst/daf/relation/sql/_engine.py

```diff
@@ -532,6 +534,7 @@ def _select_to_executable(
         `to_payload` for all other relation types.
         """
         columns_available: Mapping[ColumnTag, _L] | None = None
+        executable: sqlalchemy.sql.Select | sqlalchemy.sql.CompoundSelect
```

Comment:
Are we only supporting python 3.10 now?

Reply:
Yes, daf_relation has been 3.10-only from its inception.